src/os/bluestore/bluestore_tool.cc

```diff
@@ -291,6 +291,7 @@ int main(int argc, char **argv)
   string resharding_ctrl;
   int log_level = 30;
   bool fsck_deep = false;
+  uint64_t offset;
```

Comment:
Let's give it less specific name, like `label_offset`.

Reply:
Done